cmake/vt_add_test.cmake

```diff
@@ -23,7 +23,7 @@ associates patterns and allows configuration for common optional settings
 #]=======================================================================]
 function(vt_add_test)
     set(options)
-    set(oneValueArgs NAME)
+    set(oneValueArgs NAME MANUAL)
```

Comment:
If you are just setting this to true then this would make more sense to be an `option` than a `oneValueArgs` as you only care about its presence, not the actual value.

Reply:
Also can you document the `MANUAL` parameter in the function documentation

Reply:
Done.